gamestonk_terminal/stocks/fundamental_analysis/fa_controller.py

```diff
@@ -460,23 +457,34 @@ def call_divs(self, other_args: List[str]):
             add_help=False,
             formatter_class=argparse.ArgumentDefaultsHelpFormatter,
             prog="divs",
-            description="Get historical dividends for company",
+            description="Historical dividends for a company",
         )
         parser.add_argument(
             "-n",
             "--num",
             dest="num",
             type=check_positive,
-            default=12,
+            default=20,
```

Comment:
20 is way too much as default IMO. Can we keep this to 10?

Reply:
20 represents five years of dividend payments, assuming paid quarterly. If this number is still too high, then multiples of four work better here, IMO.

Reply:
That is a fair assumption!

Should we go with 12, 3 years?

I just think that every time I tried having 20 as default it was a bit too much. BUT in this case I was wondering if we should have dividends shown in terms of plot by default? and in that case having 20 is completely fine!